### PR TITLE
BndProjectManager: close Processor

### PR DESCRIPTION
**Pull request by jukzi**
Processor is a Closeable that was not closed.

### CODE REVIEW

**Review by laeubi — CHANGES_REQUESTED** [review]
Don't close the processor it is passed to the workspace and managed by this!

**Comment by jukzi** [comment]
As far as i can see the workspace only calls .getProperty() / getProperties() on it.

File: ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/bnd/BndProjectManager.java [hunk 1]
```diff
@@ -91,15 +91,16 @@ private static void setupProject(Project bnd, IProject project) throws CoreExcep
...
 
 	static synchronized Workspace getWorkspace() throws Exception {
-		Processor run = new Processor();
-		run.setProperty(Constants.STANDALONE, TRUE);
-		IPath path = PDECore.getDefault().getStateLocation().append(Project.BNDCNF);
-		if (workspace == null) {
-			workspace = Workspace.createStandaloneWorkspace(run, path.toFile().toURI());
-			workspace.addBasicPlugin(TargetRepository.getTargetRepository());
-			workspace.refresh();
+		try (Processor run = new Processor()) {
```
Comment (laeubi) [inline ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/bnd/BndProjectManager.java:94]: can you move the creation of the processor inside the `if (workspace == null) `

**Review by laeubi — APPROVED** [review]
Looks good

**Comment by jukzi** [comment]
ignoring random fail FeatureBasedLaunchTest https://github.com/eclipse-pde/eclipse.pde/issues/554

**Pull request merged**